docs/index.js

```diff
@@ -268,7 +268,9 @@ const App = () => {
           <Route
             component={({ location }) =>
               location.state && location.state.title ? (
-                <Helmet title={location.state.title} />
+                <Helmet title={location.state.title}>
+                  <meta name="description" content={location.state.title} />
```

Comment:
Shouldn't the meta description be different from the title?

Reply:
Updated!